tgui/packages/tgui/interfaces/PersonalCrafting.tsx

```diff
@@ -81,6 +81,7 @@ const CATEGORY_ICONS_COOKING = {
   'Mexican Food': 'pepper-hot',
   'Misc. Food': 'shapes',
   'Mothic Food': 'shirt',
+  Turkish_Food: 'shapes',
```

Comment:
```suggestion
  Turkish Food: 'shapes',
```
buranın alfabetik sırada olması gerek

Reply:
tamam bakıyorum